src/chrome/content/rules/mheducation.com.xml

```diff
@@ -74,14 +48,15 @@ Fetch error: http://careers.mheducation.com/ => https://mhecareers.silkroad.com/
 	<target host="dev.mheducation.com" />
 	<target host="highered.mheducation.com" />
 	<target host="images-cf.mheducation.com" />
-	<target host="novellaqalive.mheducation.com" />
+	<!-- <target host="novellaqalive.mheducation.com" /> -->
```

Comment:
To clean this up some, just add the non-working hosts to list above and delete these lines

Reply:
They actually happened to be hosts that no longer existed, so I just removed them.